Pipfile

```diff
@@ -12,6 +12,7 @@ flake8-docstrings = "*"
 mypy = ">=0.500"
 pytest = "*"
 pytest-cov = "*"
-mbed-targets = {path = ".", editable = true}
 pdoc3 = "*"
 toml = "*"
+mbed-targets = {path = ".",editable = true}
```

Comment:
```suggestion
mbed-targets = {path = ".", editable = true}
```

Reply:
I like this better, `pipenv` however does what I did by default for me. 🤔